### PR TITLE
Community process & handbook

### PR DESCRIPTION
**Pull request by jasonblais**

### CODE REVIEW

**Comment by jasonblais** [comment]
I think we can get this into docs and improve as we go.

File: source/process/community-guidelines.rst [hunk 1]
```diff
@@ -4,11 +4,43 @@ Mattermost Community Guidelines
...
+ 
+ - For a high priority Help Wanted ticket, reach out to contributors directly in the GitHub repository for best engagement
+
+Retention
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+1. Identify active contributors and thank them personally for their work
+2. For work in progress tickets or submitted pull requests, follow up with contributor once a week for status updates and questions. If there hasn't been activity in more than two weeks, ask if you can help them in some way
+3. Recognize everyone’s contributions (either intrinsic rewards like having their name listed on a contributor’s page or tangible like swag or gift cards)
+4. If applicable, assign roles to top contributors (such as team lead, code reviewer, tester) to give a sense of ownership for the contributor
```
Comment (lindy65) [inline source/process/community-guidelines.rst]: "...give a sense of ownership for..." > "...give a sense of ownership to..."

File: source/process/community-guidelines.rst [hunk 2]
```diff
@@ -100,3 +130,76 @@ Response Writing Tips
...
+
+  You get **10** votes in the feature idea forum, and each one influences the future of the project.
+
+Hard Questions
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To respond to tough questions, use the `SCIPAB method <https://www.mandel.com/top-ten-reasons/tools-methodology/scipab>`_ to help formulate a response and send for community lead to review. 
+
+Situation:
+  State what you know about your listeners' circumstances that is relevant to your discussion or presentation, e.g., current state of their business, technology, industry, or plans.
```
Comment (lindy65) [inline source/process/community-guidelines.rst]: "...circumstances that is relevant..." > "...circumstances that are relevant..."

File: source/process/community-overview.rst [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+
+1. You cannot attract contributors without attracting users.
+  
+  - For every 1000 users, 100 file bugs, 10 submit a patch, 1 reads the contribution guidelines
+
+2. Set realistic expectations to contributors.
+
+  - Majority of contributors will not be experts. Also, contributing to open source is typically a hobby / side job.
+
+3. There's thousands of other open source projects so it's very easy to leave to another project.
```
Comment (lindy65) [inline source/process/community-overview.rst]: "There's..." > "There are..."

File: source/process/community-overview.rst [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+
+4. Community building is mana intensive.
+
+  - You need to train contributors, dev and PM review PRs, and create recognition programs with specific roles to people.
```
Comment (lindy65) [inline source/process/community-overview.rst]: Propose replacing with:
"You need to train contributors, developers and PMs to review PRs and create recognition programs that give specific roles to people."

File: source/process/community-overview.rst [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+  - Some might build drivers to more easily integrate Mattermost into different programming languages
+  - Examples: Popular third-party software (e.g. Jira, Jenkins), Slack-compatible integrations (e.g. Slackbot), Custom Mattermost integrations (e.g. Matterbridge, Mattermail), Plug-ins (to come)
+
+2. Code contributors (Owner: Eric)
+
+  - They contribute features, small improvements and bug fixes to platform, mobile and desktop app repos
+
+3. End users (Owner: Lindy)
+
+  - They ask for help troubleshooting issues, report bugs, share UX feedback and propose new features
```
Comment (lindy65) [inline source/process/community-overview.rst]: "uservoice" > "Uservoice"

**Comment by jasonblais** [comment]
Thanks! Wondering if editor reviews for grammar should be done directly to speed up the process?

File: source/process/community-guidelines.rst [hunk 6]
```diff
@@ -4,11 +4,43 @@ Mattermost Community Guidelines
...
+
+1. Identify active contributors and thank them personally for their work
+2. For work in progress tickets or submitted pull requests, follow up with contributor once a week for status updates and questions. If there hasn't been activity in more than two weeks, ask if you can help them in some way
+3. Recognize everyone’s contributions (either intrinsic rewards like having their name listed on a contributor’s page or tangible like swag or gift cards)
+4. If applicable, assign roles to top contributors (such as team lead, code reviewer, tester) to give a sense of ownership to the contributor
+
+Off-boarding
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+1. When a campaign is finished, try to pull them to another campaign or project to work on
```
Comment (lfbrock) [inline source/process/community-guidelines.rst]: maybe say something like "offer suggestions for other campaigns or projects they might be interested in"

File: source/process/community-guidelines.rst [hunk 7]
```diff
@@ -4,11 +4,43 @@ Mattermost Community Guidelines
...
+2. Create a contribution guide including developer machine set up
+3. Create help wanted tickets that a community member could work on
+ 
+ - For a high priority Help Wanted ticket, reach out to contributors directly in the GitHub repository for best engagement
+
+Retention
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+1. Identify active contributors and thank them personally for their work
+2. For work in progress tickets or submitted pull requests, follow up with contributor once a week for status updates and questions. If there hasn't been activity in more than two weeks, ask if you can help them in some way
```
Comment (lfbrock) [inline source/process/community-guidelines.rst]: maybe phrase this around helping make sure contributors aren't blocked with questions

**Pull request merged**